docs/api.md

```diff
@@ -212,6 +212,17 @@ gef_on_exit_unhook
 > GDB are only present in the very latest version of GDB.
 
 
+```
```

Comment:
We don't do it other places, but we can put `python` here for syntax highlighting.

Reply:
Sure